lib/engines/content_object_store/app/workers/content_object_store/schedule_publishing_worker.rb

```diff
@@ -48,9 +48,11 @@ def perform(edition_id)
 
       schema = ContentObjectStore::ContentBlock::Schema.find_by_block_type(edition.document.block_type)
 
-      ContentObjectStore::PublishEditionService.new(
-        schema,
-      ).call(edition)
+      ContentObjectStore::HasAuditTrail.acting_as(publishing_robot) do
```

Comment:
this is the "publishing robot" used by Whitehall, and called the "Scheduled Publishing Robot" - should we create our own User for this? Not entirely sure of the best way of doing that in Int/Prod...

Reply:
I think let's keep it the same, as, to a user, publishing is the same regardless of where in Whitehall it's done.